dlc-oracle/src/main/scala/org/bitcoins/dlc/oracle/DLCOracleAppConfig.scala

```diff
@@ -114,7 +114,8 @@ case class DLCOracleAppConfig(
                                           aesPasswordOpt,
                                           bip39PasswordOpt)
     val oracle = DLCOracle(key)(this)
-    initialize(oracle)
+
+    start().map(_ => oracle)
```

Comment:
Is it possible to to start the `DLCOracleAppConfig` with calling `initialize()`?  

Also more generally, is there a reason we don't just use all our existing code in the key manager project rather than copy pasting some of it over to here?

Reply:
> Is it possible to to start the DLCOracleAppConfig with calling initialize()?

That's what this is doing.

> Also more generally, is there a reason we don't just use all our existing code in the key manager project rather than copy pasting some of it over to here?

Fixed